hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -51,6 +53,15 @@ def __init__(self) -> None:
         self._err_bits = 0
         self.pending_halt = False
 
+        self._new_rnd_data = None
+        self._urnd_reseed_complete = False
+
+    def set_rnd_data(self, rnd_data: int) -> None:
+        self._new_rnd_data = rnd_data
+
+    def set_urnd_reseed_complete(self) -> None:
+        self._urnd_reseed_complete = True
```

Comment:
I don't have really strong feelings about this, but setters and getters like this look a bit too much like Java for my taste. Maybe it's better just to drop the `_` prefixes from the fields and set them explicly?

Reply:
I have a preference for them where you only need one of set or get. Here a user of class shouldn't be reading `_new_rnd_data` or `_urnd_reseed_complete` nor need to worry about their values in general and I think the use of the setters helps emphasise this (being Python you can just ignore this and read the values directly of course).